packages/snap-preact/src/Instantiators/RecommendationInstantiator.tsx

```diff
@@ -248,6 +248,12 @@ export class RecommendationInstantiator {
 					await controller.search();
 				}
 
+				if (!controller.store.results.length && controller.store.error) {
+					//something went wrong
+					//err was already logged - nothing to do.
+					return;
```

Comment:
I don't think we want to return here - as there are several things that happen after this. For example, adding the controller to the instantiator list of controllers, putting the controller in the window object, etc...